### DISCUSSION TITLE
LobeHub 的midjourney_proxy_url

### DISCUSSION THREAD

**Discussion by toyutim** [lobehub/lobehub#2448] [Development Question]
我已经打了https://discord.com/channels/XXX/XXXX 到midjourney_proxy_url，然后在lobechat使用Midjourney 插件,打了Prompt，然后打圈运算了好久，一直都没有反应，有没有大神都出现这情况 ? 可以解决吗? 谢谢大神!!

**Comment by yanghanlin** [comment]
[文档](https://github.com/lobehub/lobe-midjourney-webui/tree/main#:~:text=This%20plugin%20relies%20on%20midjourney%2Dproxy%20as%20the%20provider%20for%20Midjourney%20services.%20You%20will%20need%20to%20deploy%20a%20midjourney%2Dproxy%20service.)中似乎是说需要部署一个 [midjourney-proxy](https://github.com/novicezk/midjourney-proxy) 实例，再把实例 API 地址填入 `MIDJOURNEY_PROXY_URL`
